hook.js

```diff
@@ -9,68 +10,75 @@ var parseSource = require('./sourceParse');
 
 var token = 'token ' + process.env.OAUTH_TOKEN;
 var productName = 'Discord';
```

Comment:
Just following the same convention used elsewhere. For example, we have a repository called [bedrock](https://github.com/mozilla/bedrock/) that powers a product named [Bedrock](https://wiki.mozilla.org/Mozilla.com/Bedrock). Same with Kuma and others.